zerver/openapi/zulip.yaml

```diff
@@ -12369,22 +12373,26 @@ paths:
 
                               The value needs to be a standard language code that the Zulip server has
                               translation data for; for example, `"en"` for English or `"de"` for German.
-                          default_view:
+                          web_home_view:
                             type: string
                             description: |
-                              The [default view](/help/configure-home-view) used when opening a new
+                              The [home view](/help/configure-home-view) used when opening a new
                               Zulip web app window or hitting the `Esc` keyboard shortcut repeatedly.
 
                               - "recent_topics" - Recent conversations view
                               - "inbox" - Inbox view
                               - "all_messages" - All messages view
-                          escape_navigates_to_default_view:
+
+                              **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
+                              called `default_view`.
+                          web_escape_navigates_to_home_view:
                             type: boolean
                             description: |
                               Whether the escape key navigates to the
-                              [configured default view](/help/configure-home-view).
+                              [configured home view](/help/configure-home-view).
 
-                              **Changes**: New in Zulip 5.0 (feature level 107).
+                              **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
+                              called `escape_navigates_to_default_view`.
```

Comment:
Do we also need to mention the feature level in which the original `escape_navigates_to_default_view` was added?

Reply:
Yep, fixed with this diff. Note also that we always write `Previously, ...` as a small English grammar detail.

``` diff
diff --git a/zerver/openapi/zulip.yaml b/zerver/openapi/zulip.yaml
index 6f56cbadb6..bba6d0d717 100644
--- a/zerver/openapi/zulip.yaml
+++ b/zerver/openapi/zulip.yaml
@@ -10042,8 +10042,8 @@ paths:
             - "inbox" - Inbox view
             - "all_messages" - All messages view
 
-            **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-            called `default_view`.
+            **Changes**: New in Zulip 8.0 (feature level 219). Previously, this was
+            called `default_view`, which was new in Zulip 4.0 (feature level 42).
           schema:
             type: string
           example: all_messages
@@ -10053,8 +10053,8 @@ paths:
             Whether the escape key navigates to the
             [configured home view](/help/configure-home-view).
 
-            **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-            called `escape_navigates_to_default_view`.
+            **Changes**: New in Zulip 8.0 (feature level 219). Previously, this was called
+            `escape_navigates_to_default_view`, which was new in Zulip 5.0 (feature level 107).
           schema:
             type: boolean
           example: true
@@ -12383,16 +12383,17 @@ paths:
                               - "inbox" - Inbox view
                               - "all_messages" - All messages view
 
-                              **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-                              called `default_view`.
+                              **Changes**: New in Zulip 8.0 (feature level 219). Previously, this was
+                              called `default_view`, which was new in Zulip 4.0 (feature level 42).
                           web_escape_navigates_to_home_view:
                             type: boolean
                             description: |
                               Whether the escape key navigates to the
                               [configured home view](/help/configure-home-view).
 
-                              **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-                              called `escape_navigates_to_default_view`.
+                              **Changes**: New in Zulip 8.0 (feature level 219). Previously, this
+                              was called `escape_navigates_to_default_view`, which was new in Zulip
+                              5.0 (feature level 107).
                           left_side_userlist:
                             type: boolean
                             description: |
@@ -13241,8 +13242,8 @@ paths:
                           See [PATCH /settings](/api/update-settings) for details on
                           the meaning of this setting.
 
-                          **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-                          called `default_view`.
+                          **Changes**: New in Zulip 8.0 (feature level 219). Previously, this was
+                          called `default_view`, which was new in Zulip 4.0 (feature level 42).
 
                           **Deprecated** in Zulip 5.0 (feature level 89). Clients connecting to newer
                           servers should declare the `user_settings_object` client capability and
@@ -14583,16 +14584,17 @@ paths:
                               - "inbox" - Inbox view
                               - "all_messages" - All messages view
 
-                              **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-                              called `default_view`.
+                              **Changes**: New in Zulip 8.0 (feature level 219). Previously, this was
+                              called `default_view`, which was new in Zulip 4.0 (feature level 42).
                           web_escape_navigates_to_home_view:
                             type: boolean
                             description: |
                               Whether the escape key navigates to the
                               [configured home view](/help/configure-home-view).
 
-                              **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-                              called `escape_navigates_to_default_view`.
+                              **Changes**: New in Zulip 8.0 (feature level 219). Previously, this
+                              was called `escape_navigates_to_default_view`, which was new in Zulip
+                              5.0 (feature level 107).
                           left_side_userlist:
                             type: boolean
                             description: |
@@ -15652,8 +15654,8 @@ paths:
             - "inbox" - Inbox view
             - "all_messages" - All messages view
 
-            **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-            called `default_view`.
+            **Changes**: New in Zulip 8.0 (feature level 219). Previously, this was
+            called `default_view`, which was new in Zulip 4.0 (feature level 42).
 
             Before Zulip 5.0 (feature level 80), this setting was managed by
             the `PATCH /settings/display` endpoint.
@@ -15668,8 +15670,9 @@ paths:
             Whether the escape key navigates to the
             [configured home view](/help/configure-home-view).
 
-            **Changes**: New in Zulip 8.0 (feature level 219). Previously this was
-            called `escape_navigates_to_default_view`.
+            **Changes**: New in Zulip 8.0 (feature level 219). Previously, this
+            was called `escape_navigates_to_default_view`, which was new in Zulip
+            5.0 (feature level 107).
           schema:
             type: boolean
           example: true
```